index.html

```diff
@@ -40,7 +40,7 @@ <h1>no</h1>
 <script>
     const isToday = () => {
         // 12 feb 2020
-        console.log(1)
+        // console.log(1)
```

Comment:
no

Reply:
// no

Reply:
There has been a major update on this feature as of this post - I appreciate the work you have put into this and can't imagine it's a big job to work this feature into the newer version and would urge you to do so over the coming weeks.